getting-started/compress-data.md

```diff
@@ -0,0 +1,161 @@
+# Compression policies
+TimescaleDB includes native compression capabilities which enable you to
+analyze and query massive amounts of historical time-series data inside a
+database while also saving on storage costs. Additionally, all PostgreSQL data
+types can be used in compression.
+
+Compressing time-series data in a hypertable is a two-step process. First, you
+need to enable compression on a hypertable by telling TimescaleDB how to compress
+and order the data as it is compressed. Once compression is enabled, the data can
+then be compressed in one of two ways:
+
+* Using an automatic policy
+* Manually compressing chunks
+
+## Enable TimescaleDB compression on the hypertable
+
+To enable compression, you need to [`ALTER` the `stocks_real_time` hypertable][alter-table-compression]. There
```

Comment:
```suggestion
To enable compression, you need to [`ALTER`][alter-table-compression] the `stocks_real_time` hypertable. There
```
Because links that span multiple inline code sections look ugly: the underline keeps stopping and starting.